qemu.mk

```diff
@@ -79,10 +79,12 @@ busybox-cleaner: busybox-cleaner-common
 ################################################################################
 # Linux kernel
 ################################################################################
-$(LINUX_PATH)/.config:
+$(LINUX_PATH)/.config: kconfig_preempt.conf
```

Comment:
You're right, my bad! For some reason I was just thinking it was added to the kernel dir.